test/test_ha_mqtt_client.py

```
@@ -0,0 +1,14 @@
+from camvidlog2.ha_mqtt_client import MQTTClient
+
+
+def test_basic(mqtt_broker):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_basic`

Add return type annotation: `None`

(ANN201)